src/content/basics/security/index.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: Security
 description: Documentation of the Giant Swarm cluster security
-date: 2020-03-12
+date: 2020-05-18
 type: page
 weight: 20
 categories: ["basics"]
-last-review-date: 2020-03-12
+last-review-date: 2020-05-18
 ---
 
 # Security
 
-This reference gives you details on security-related measures in a Giant Swarm installation.
+Regardless of provider, clusters start with restrictive settings already in place. All internal and external network traffic is denied by default, role-based access control prevents unauthorized access to Kubernetes resources, and Pod security policies are in place to prevent Pods from running with root users, risky volume configurations, or additional privileges. All of these can be adapted by the customer to fit their needs. 
 
-## Encryption
+We regularly test our configurations against the CIS Kubernetes Benchmark as well as the CNCF Certified Kubernetes Conformance standard to ensure our platform remains compliant with industry best-practices.
```

Comment:
```suggestion
We regularly test our configurations against the CIS Kubernetes Benchmark as well as other CIS and industry benchmarks for Docker, Linux, and cloud providers to ensure our platform remains compliant with industry best-practices.
```

We focus on CIS K8s but we also do others when working with customers. We have even done a few independent pen tests, but we do those very irregularly, or rather bigger customers do them in the first year of engagements.